### PR TITLE
Added Rackspace support for DE

### PR DESCRIPTION
**Pull request by justnoxx**

### CODE REVIEW

File: pages/EC-OpenStack_help.xml [hunk 1]
```diff
@@ -1726,7 +1726,10 @@ outline used above where possible.
 
 
     <h1 id="rns">Release Notes</h1>
-
+    <h2>@PLUGIN_KEY@ 1.3.4</h2>
+    <ul>
+      <li>Added Rackspace support for Dynamic Environments</li>
+    </ul>
       <h2>@PLUGIN_KEY@ 1.3.3</h2>
```
Comment (urvashisingh) [inline pages/EC-OpenStack_help.xml]: We haven't published/released 1.3.3 yet so we can keep the same version number.

File: src/main/resources/project/ui_forms/deploy_DE.xml [hunk 2]
```diff
@@ -87,6 +87,16 @@
...
+        <label>Region:</label>
+        <property>region</property>
+        <value/>
+        <documentation>The region zone in which to launch the server.</documentation>
+        <required>0</required>
+        <type>entry</type>
+        <serverOptions>0</serverOptions>
+        <dependsOn>connection_config</dependsOn>
+    </formElement>
+    <formElement>
```
Comment (urvashisingh) [inline src/main/resources/project/ui_forms/deploy_DE.xml]: Between availability zone and region, is at least one of them required? If so, we should add it to the validations for dynamic envts. 
We also need to add the drop-down for regions and have it be populated dynamically similar to availability zones.

File: src/main/resources/project/project.xml [hunk 3]
```diff
@@ -1420,7 +1436,14 @@
                 <formalParameterName>availability_zone</formalParameterName>
                 <defaultValue>1</defaultValue>
                 <description/>
-                <required>1</required>
+                <required>0</required>
+                <type>entry</type>
+            </formalParameter>
+            <formalParameter>
+                <formalParameterName>region</formalParameterName>
+                <defaultValue>1</defaultValue>
```
Comment (urvashisingh) [inline src/main/resources/project/project.xml]: Why do we have a default value for region?

Comment (justnoxx) [inline src/main/resources/project/project.xml]: Because availability_zone also has default value = 1. And all other optional fields too.

**Pull request merged**